components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -261,12 +261,12 @@ export default function () {
                 )}
             </div>
             <div>
-                <h3 className="mt-12">Delete Project</h3>
+                <h3 className="mt-12">Remove Project</h3>
                 <p className="text-base text-gray-500 dark:text-gray-400 pb-4">
-                    Removing the project from this team will also remove team members access to it.
+                    Removing this project will also remove all associated data with this project, including workspaces.
```

Comment:
Sorry, I might have confused you with a previous comment. We do not remove (delete) any associated data atm. I was just wondering if we should and/or what the consequences of deleting a project are.

Reply:
Ah my mistake, I copied this wording from another related issue. I'll adjust this specific text back to how it was. Thanks for catching.